build_latest_snapshot

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
one last thing: since this script takes an optional explicit snapshot id, let's not use "latest" in the name (it would be confusing)

let's just rename it to `buildone_snapshot`

Reply:
done!